src/test/scala/com/github/mrpowers/spark/fast/tests/DataFrameComparerTest.scala

```diff
@@ -203,18 +197,11 @@ class DataFrameComparerTest extends FreeSpec with DataFrameComparer with SparkSe
         List(("number", IntegerType, true))
       )
 
-      val e = intercept[DatasetContentMismatch] {
+      val _ = intercept[DatasetContentMismatch] {
         assertLargeDataFrameEquality(sourceDF, expectedDF)
       }
-      val e2 = intercept[DatasetContentMismatch] {
```

Comment:
Just to be sure, deleting these lines is because they use the same functionality in both methods, isn't it?

Reply:
I have added a different set of test for assertLargeDataFrameEquality and keep the original set for assertSmallDataFrameEquality. Maybe there is a better way to parameterize these test without having to duplicate the test case if we are sure that both functions will always have the same signature

Reply:
We can see this in other PR to not mix so many things in one single pr

Reply:
Sure. Let do those in another PR then. I have revert the test and remove those additional one